modules/distribution/product/src/main/resources/conf/infer.json

```diff
@@ -7,11 +7,9 @@
       "apim.throttling.enable_blacklist_condition": true,
       "indexing.enable": false,
       "system.parameter.profile": "gateway-worker",
-      "enabled_global_handlers": [
-          "external_call_logger"
-        ],
-       "synapse_handlers.external_call_logger.name": "externalCallLogger",
-       "synapse_handlers.external_call_logger.class": "org.wso2.carbon.apimgt.gateway.handlers.LogsHandler"
+      "enabled_global_handlers": ["external_call_logger"],
+      "synapse_handlers.external_call_logger.name": "externalCallLogger",
+      "synapse_handlers.external_call_logger.class": "org.wso2.carbon.apimgt.gateway.handlers.LogsHandler"
```

Comment:
seems no changes here and just formatting. can you revert it?

Reply:
oh, ok.